go.sum

```diff
@@ -2,6 +2,8 @@ github.com/bitcoin-sv/go-sdk v1.1.14 h1:65PrWc8H5in138fplswpxpXuV3HwQQgyXDBegioy
 github.com/bitcoin-sv/go-sdk v1.1.14/go.mod h1:khREs6RkJuYkxFD3ZB9I+zcGjfxoHeACg9Zr3f7Fn7s=
 github.com/bitcoin-sv/spv-wallet/models v1.0.0-beta.32 h1:YXiGH7gNKSR7qsLrOiO6zeWmGsA0RynNWeKJ0g5QRBk=
 github.com/bitcoin-sv/spv-wallet/models v1.0.0-beta.32/go.mod h1:PEJdH9ZWKOiKHyOZkzYsRbKuZjzlRaEJy3GsM75Icdo=
+github.com/bitcoin-sv/spv-wallet/models v1.0.0-beta.33 h1:hKB8uen+pdFjt/zXdv42JotRihyaXXZ+vxxrsIxKPEY=
+github.com/bitcoin-sv/spv-wallet/models v1.0.0-beta.33/go.mod h1:PEJdH9ZWKOiKHyOZkzYsRbKuZjzlRaEJy3GsM75Icdo=
```

Comment:
Why in `go.sum` there is also `-beta.32` included? 🤔 
Have you run `go mod tidy`?

Reply:
I thought I did 😰 but apparently didn't good catch

Reply:
now i did :D